lib/wallet/fund_wallet_on_chain.dart

```diff
@@ -0,0 +1,129 @@
+
+class FundWalletOnChain extends StatefulWidget {
+  const FundWalletOnChain({Key? key}) : super(key: key);
+
+  static const route = '/' + subRouteName;
+  static const subRouteName = 'fund-wallet-on-chain';
+
+  @override
+  State<FundWalletOnChain> createState() => _FundWalletOnChainState();
+}
+
+class _FundWalletOnChainState extends State<FundWalletOnChain> {
+  Network network = Network.Testnet;
+
+  @override
+  void initState() {
+    _callInit();
+    super.initState();
+  }
+
+  Future<void> _callInit() async {
+    try {
+      final network = await api.network();
+      setState(() {
+        this.network = network;
+      });
+    } on FfiException catch (error) {
+      FLog.error(text: "Failed to fetch address phrase: Error: " + error.message, exception: error);
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Fund Wallet'),
+      ),
+      body: SafeArea(
+          child: Container(
+        padding: const EdgeInsets.all(20.0),
+        child: Column(
+          children: [
+            SizedBox(
+                child: Container(
+              margin: const EdgeInsets.all(20),
+              padding: const EdgeInsets.all(10),
+              decoration: BoxDecoration(
+                color: Colors.orange,
+                borderRadius: BorderRadius.circular(100),
+              ),
+              child: const RotatedBox(
+                quarterTurns: 1,
+                child: IconButton(
+                  icon: Icon(
+                    FontAwesomeIcons.arrowRightToBracket,
+                    color: Colors.white,
+                  ),
+                  onPressed: null,
+                ),
+              ),
+            )),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: const [
+                Text("Fund your wallet",
+                    style: TextStyle(fontWeight: FontWeight.w600, fontSize: 20)),
+              ],
+            ),
+            const SizedBox(height: 10),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: const [
+                Flexible(
+                  child: Text(
+                    "Get bitcoin from our faucet or deposit on-chain to get started.",
+                  ),
+                ),
+              ],
+            ),
+            const SizedBox(height: 10),
+            Expanded(
+              child: ListView(
+                children: [
+                  const Divider(),
+                  InkWell(
+                      onTap: () {
+                        Navigator.push(
+                          context,
+                          MaterialPageRoute(builder: (context) => const DrainFaucet()),
+                        );
+                      },
+                      child: const ListTile(
+                        title: Text("Drain faucet"),
```

Comment:
🙃 This might just be me, but drain makes me think that you are going to take all of the coins haha. My suggestion is just an alternative that I don't love either.

```suggestion
                        title: Text("Use faucet"),
```

Reply:
If we change `drain` to something else it would be great to update it throughout - we use it in the route as well (...)